src/components/FileRepo/AggregationSidebar/aggsConfig.js

```diff
@@ -33,11 +36,11 @@ export const aggsConfig = (data, graphqlField) => {
   const gqlAggregationFields = types.find(({ name }) => name === `${graphqlField}Aggregations`)
     .fields;
 
-  const typeAggsConfig = type =>
-    type.map(fieldName => ({
+  const typeAggsConfig = (type) =>
+    type.map((fieldName) => ({
       field: fieldName,
       show: true,
-      type: gqlAggregationFields.find(fileAggField => fieldName === fileAggField.name).type.name,
+      type: gqlAggregationFields.find((fileAggField) => fieldName === fileAggField.name).type.name,
```

Comment:
I don't know if you want to take the opportunity to rename 'type'. 
Just the fact that it's singular, so how can a `.map` work on it.

Reply:
Good point. I would add that .find(fileAggField => fieldName === fileAggField.name).type.name, could be extracted in 1 func